.travis.yml

```diff
@@ -4,16 +4,11 @@ cache: pip
 
 python:
   - "3.5"
-  - "2.7"
 
 env:
   - DJANGO="django>=1.11,<1.12"
-  - DJANGO="django>=2.0,<2.2"
+  - DJANGO="django>=2.0,<2.3"
```

Comment:
try changing this to `DJANGO="django>=2.2,<2.3"`

Reply:
Done :smile: